elm-package.json

```diff
@@ -4,7 +4,7 @@
     "repository": "https://github.com/user/project.git",
     "license": "BSD3",
     "source-directories": [
-        "elm/"
+        "elm/", "../elm-css/src"
```

Comment:
Yeah, elm-css doesn't expose all the modules needed right now. So this is a placeholder til that's fixed upstream